package-lock.json

```diff
@@ -4110,7 +4110,8 @@
         "ansi-regex": {
```

Comment:
This file doesn't need to be updated since there were no dependency updates.

Reply:
yeah not sure why...